text/0000-dll-kind.md

```diff
@@ -0,0 +1,107 @@
+- Feature Name: dll_kind
```

Comment:
s/dll_kind/kind_raw_dylib/ (both here and in the filename) please.

Reply:
I did the thing.